octobot_trading/channels/orders.py

```diff
@@ -30,8 +30,7 @@ class OrdersProducer(ExchangeChannelProducer):
     async def perform(self, orders, is_closed=False, is_from_bot=True):
         try:
             for order in orders:
-                symbol: str = self.channel.exchange_manager.get_exchange_symbol(
-                    order[ExchangeConstantsOrderColumns.SYMBOL.value])
+                symbol: str = order[ExchangeConstantsOrderColumns.SYMBOL.value]
```

Comment:
Is it not dangerous to use directly the order symbol ? I'm affraid it can create incompatibilities for symbols that rebrand (ex: XRB raiblocks => NANO nano) that are sometimes still refered as XRB. Originally ```get_exchange_symbol``` was here to handle these

Reply:
It's already ccxt normalized (the ccxt equivalent of `get_exchange_symbol` has already been called) here but we can keep the call to ccxt if you prefer.

Reply:
If you are sure it can never (with the current code) create conflicts, then we should remove it

Reply:
What do you mean by creating conflicts ? You mean having an unexpected symbol, something that hasn't been normalized by ccxt before entering `perform` ?

Reply:
Exactly, ex: a user wants to trade NANO/BTC, it's translated into XRB/BTC under the hood, it will come back as XRB/BTC, if it's not converted back into NANO/BTC, it will create issues. Same with BTC that is not called BTC on every exchange, etc

Reply:
Yes ok, it's a good security to keep. I'll rollback this changes.